time stamp
int32 msg_id
float32 theta_m_update
float32 theta_l_update
bool should_reset
---
bool ack
